packaging/linux/provision.sh

```diff
@@ -104,7 +78,5 @@ if which yum; then
     build_rpm && rpm_result="success"
 fi
 
-# this should be used AFTER renaming the cli packages to contain versions.
-# cd /tmp/x86_64 && md5sum=$(md5sum *.rpm) && echo $md5sum | sudo tee ${md5sum##* }.md5 &&
-[ "$rpm_result" == "success" ] && cd /tmp/x86_64 && md5sum=$(md5sum *.rpm) && echo $md5sum | sudo tee ${md5sum##* }.md5 &&
-[ -z ${AWS_ACCESS_KEY} ] || upload_to_s3
+[ "$rpm_result" == "success" ] && cd /tmp/x86_64 && create_md5 "rpm" && [ -z ${AWS_ACCESS_KEY} ] ||
```

Comment:
can you please put each of the commands in its own line so that it's readable?

thanks